composer.json

```diff
@@ -902,7 +902,7 @@
     "symfony/web-profiler-bundle": "^4.0.0",
     "codeception/module-rest": "^1.0",
     "codeception/module-phpbrowser": "^1.0.1",
-    "codeception/module-asserts": "^1.2.1",
+    "codeception/module-asserts": "^1.3.0",
```

Comment:
changes from commit are missung https://github.com/spryker/suite-nonsplit/commit/e71ea9a8c951ab8e5f27a3421d3dc53313487d0a

Reply:
even more https://github.com/spryker/suite-nonsplit/pull/4459/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R32

Reply:
Can you add those on top?
I pushed the missing dependency.

Reply:
I dont think the dependencies themselves are valid to be added though
The conflict resolution preferring "own" (split) is correct here, as those are dependencies of a split module usually, and then come in as such ( no need to have those in root level ).